seed/seed.json

```diff
@@ -1219,6 +1219,27 @@
                 "channel": ["NIGHTLY"],
                 "platform": ["WINDOWS", "MAC", "LINUX"]
             }
+        },
+        {
+            "name": "BraveScreenFingerprintingBlockerRollout",
+            "experiments": [
+                {
+                    "name": "Enabled",
+                    "probability_weight": 50,
+                    "feature_association": {
+                        "enable_feature": ["BraveBlockScreenFingerprinting"]
+                    }
+                },
+                {
+                    "name": "Default",
+                    "probability_weight": 50
+                }
+            ],
+            "filter": {
+                "min_version": "105.0.5195.68",
```

Comment:
this should be Brave version which takes first value from Chromium version and other values from Brave version: `105.1.45.38`.

Reply:
Ah I missed that. Thanks!